### ISSUE TITLE
Use #cycles listed in runParameters.xml

### ISSUE THREAD

**Issue by ingkebil** [Clinical-Genomics/demultiplexing#3]
HO runs have different # cycles than RM runs. We should read the #cycles from the runParameters.xml file:

```
<Read1>126</Read1>
<IndexRead1>8</IndexRead1>
<IndexRead2>0</IndexRead2>
<Read2>126</Read2>
```

**Comment by ingkebil** [comment]
Actually .. you can also use RunInfo.xml for that.

**Comment by ingkebil** [comment]
Fixed in cda032154e5ebe71
